### DISCUSSION TITLE
How to disable this annoy red dot (●) every time there are changes in git repository

### DISCUSSION THREAD

**Discussion by hzung** [ohmyzsh/ohmyzsh#10800] [Help] [answered]
![image](https://user-images.githubusercontent.com/12733328/160221881-f6143e64-405f-454e-affe-f2b8c81104e0.png)

I understand that this feature is useful for someone. But I feel really uncomfortable when I see some red dot like this. How to config to disable it?

Thank you.

**Comment by rwmitchell** [comment]
Which theme are you using?

**Reply by hzung** [reply]
> Which theme are you using?

ZSH_THEME="bira"

**Comment by rwmitchell** [comment, marked as answer]
It appears to be coming from this line in the theme:

```sh
ZSH_THEME_GIT_PROMPT_DIRTY="%{$fg[red]%}●%{$fg[yellow]%}"
```

Either unset this variable, change it to something you do like, or pick a new theme.
